.github/actions/check_cirrus_cron/cron_failures.sh

```diff
@@ -118,5 +85,5 @@ cat "$NAME_ID_FILEPATH"
 records=$(wc --words "$NAME_ID_FILEPATH" | cut -d ' ' -f 1)
 # Always two words per record
 failures=$((records/2))
-echo "::set-output name=failures::$failures"
+echo "failures::$failures" >> $GITHUB_OUTPUT
```

Comment:
This is a good point.  I should add a comment referencing https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Basically, Github-actions deprecated the former way of setting output values (forcing an update to every ^*#@bleep0]!n1_! action and workflow in the universe).  So this is the new standard.  The script assumes basic knowledge of github-actions, and so that's why it's not commented.  But it's an easy thing to add.